subgraphs/aave-forks/protocols/aave-amm/src/mapping.ts

```diff
@@ -350,21 +288,76 @@ export function handleRepay(event: Repay): void {
     event,
     event.params.amount,
     event.params.reserve,
-    getProtocolData(),
+    protocolData,
     event.params.user
   );
 }
 
 export function handleLiquidationCall(event: LiquidationCall): void {
+  // if liquidator chooses to receive AToken, create a position for liquidator
+  let createLiquidatorPosition = false;
```

Comment:
I am struggling to find in the aave contracts.

But when a `createLiquidatorPosition` is true does it send out a `BalanceTransfer` event? I am asking for my understanding and to make sure we are not double counting anything.

Reply:
good catch! I believe there is a Transfer (and BalanceTransfer) event emitted when the AToken in the liquidation is transferred from liquidatee to liquidator. So we don't ever need to set createLiquidatorPosition to true for aave-forks. fixed in https://github.com/messari/subgraphs/pull/1993/commits/e3100a3be93625a2b3f379bead26bb9c85fed253

Reply:
If `createLiquidatorPosition` is always false, why leave the logic in there?

Reply:
good point! It is not needed for aave-forks. Do you think it makes sense to merge this upstream to the sdk, as it may be useful for other protocols?

Reply:
If there is a standard way you can implement, I am weary about there being a standard implementation. Since liquidations may pay in market output tokens or in the input token it adds more complexity

Reply:
ok, will remove.